Makefile

```diff
@@ -1,72 +1,53 @@
-all: sysl
-
-input = api/project.sysl
-app = simple
-dependencies = jsonplaceholder # this can be a list separated by a space or left empty
-outdir = gen
-# Current go import path
-basepath = github.com/anz-bank/sysl-template
```

Comment:
This Makefile used to be structured such that teams that clone the template would only have to concern themselves with the top few lines. In the new structure, it seems they would have to read through the whole thing and find the places they need to change, and maintain it going forward.

Is that the intent? And is there some way to make it more user-friendly for users that don't like `make`?

Reply:
The Makefile earlier was more suitable to the repository being used as a template for new apps which use sysl codegen. The proposed changes however adapt this project to serve as a Sysl-Go demo and not exactly as a template and the project itself has become more complicated by having 1 App and 2 Downstream Systems. I will still see if I could provide a few levers in the makefile which the developers could modify and reuse the implementation.